### ISSUE TITLE
Станция мини 2 с часами

### ISSUE THREAD

**Issue by Opontios** [AlexxIT/YandexStation#310]
Некоторое время (к сожалению не удалось проследить с какими обновлениями было связано, но точно что появилось относительно недавно) наблюдается следующее ненормальное поведение у колонок с часами. Уровень громкости не синхронизируется в карточке с реальным уровнем при смене голосовой командой или кнопками на колонке, и при обновлении страницы HA в браузере уровень всегда переходит на 50%. Но еще более раздращающее то, что при любом уровне громкости если выполнается скрипт с данными, например:
`
service: media_player.play_mediа
data:
    media_content_id: Ну наконец-то ты пришел!
    media_content_type: text
    extra:
        volume_level: 0.8
target:
   entity_id: 
       - media_player.yandex_station_m00m0s000wgryk
       - media_player.yandex_station_lr000000000000017613000053554faa`
Или такой:
`
service: tts.yandex_station_say
   entity_id: media_player.yandex_station_m00m0s000wgryk
   data:
      message: Ну наконец-то ты пришел!
      options:
          volume_level: 0.8`
После выполнения скрипта уровень громкости на станции "лайт" или "мини" возвращается на предыдущий, а на станциях "мини 2 с часами" переходит на 50% всегда. Для исключения какого-либо глюка несвязанного с самой колонкой распаковал новую мини с часами, подключил к новому аккаунту Яндекс, обновил, установил новый HA instance на VM, и только интеграцию для колонки. Проблема воспросизводится.

**Comment by Opontios** [comment]
`Некотор

рое
время
так`

**Comment by Opontios** [comment]
Уважаемый AlexxIT, eсть ли какая-то надежда что удасться поправить? У меня куча скриптов в автоматизациях с TTS, и в виду того, что это новая модель, поменял все колонки дома на эти. И именно с ними проблема.

**Comment by AlexxIT** [comment]
Я планировал посмотреть на своей. Пока не могу подтвердить или опровергнуть ошибку.

**Comment by ghork125** [comment]
Подобная ситуация и у меня, только у меня была проблема на 2х колонках мини 1го поколения и на 2х мини2 с часами.
Уровень звука из HA применяется к колонке, но в атрибутах всегда звук стоит 0,5 и не возможно поменять атрибут is_volume_muted, я так и проверял на скорую руку работает ли нормально колонка, если звук не вырубается по mute то значит проблема все еще есть. При этом треки, обложки, позицию в песне - все показывает правильно.
В это же время Яндекс.Станция работала в штатном режиме.
Спустя какое то время получилось решить проблему на 3х колонках. Для этого заходил в устройства и в медиа плеере менял ID объекта просто добавив к имени 1 в конце, через какое то время дерганья уровня звука начинало все работать. Затем менял ID объекта назад - колонки работали как и раньше. Но 1 из 2х мини2 с часами отказывается нормально работать. Прошивки на обоих мини2 одинаковые - 0.134.1.28.1425641459.20220826.79.
Отвязывание колонки в Яндексе, Удаление интеграции из HA - ничего не помогает.

**Comment by ghork125** [comment]
Проблема вернулась часов через 12, теперь 1 мини и 1 мини2 работают как положено и 1 мини и 1 мини2 так же неадекватно себя ведут, звук всегда остается 0.5 в HA и mute не работает.

**Comment by Opontios** [comment]
> Проблема вернулась часов через 12, теперь 1 мини и 1 мини2 работают как положено и 1 мини и 1 мини2 так же неадекватно себя ведут, звук всегда остается 0.5 в HA и mute не работает.

Именно. Ничего из перечисленного Вами не помогает. К тому же, если до этого проблема проявлялась только на "мини 2 с часами", с сегодняшнего дня присоединилась и "лайт". Пока не смог проверить что с "мини" первого поколения, но судя по тому, что происходит, дело в обновлениях, которые приходят на колонки, и ломают интеграцию Yandex.station.

**Comment by Opontios** [comment]
@ghork125 
Судя по всему только у нас двоих такая проблема. Или же это никому не мешает.

**Comment by KAPACb1988** [comment]
Здравствуйте. Тоже есть проблема со звуком.. две колонки мини2 без часов.
Сначала заметил, что с одной проблемы.. если смотреть через разработчика, состояния. volume_level: 0.5 и этот атрибут не синхронизируется.. поменяв звук кнопками с колонки, звук становится громче или тише, но атрибут остается 0.5. другие параметры синхронизируются.
в lovelace я добавил mediaplayer, кнопки добавить громкость и убавить не работают.. а если точнее, когда нажимаешь + звук становится громче, потом при нажатии на + он всегда на одном уровне... если нажать звук -, звук становится тише, потом также если еще нажать - на одном уровне. Атрибут при этом не меняется, стоит 0.5. Если крутить звук не кнопками, а дорожкой (держим левую кнопку мыши и тянем на нужный нам уровень громкости), громкость изменяется, но атрибут как я уже говорил 0.5 всегда. Я так понял не работает служба mediaplayer volume down и volume up, при этом set volume работает.
еще наблюдение.. вторая колонка работала нормально!! и атрибут менялся и кнопки - и + работали, но только стоило мне ее обесточить и подключить в розетку заново, появилась такая же проблема как и с первой колонкой.
еще наблюдение (но не помню что я делал, толи id поменял, толи при добавлении интеграции это случилось).. 
выключил из розетки, включил, увидел в lovelace медиаплеер не как яндекс станцию по виду (с обложкой), а без обложки песни, попробовал + или - нажать (уже не помню) и слышу, Алиса говорит "этот уровень громкости уже установлен" и потом при нажатии кнопок + и - звук менялся, но не как обычно быстро, а с какой - то задержкой (секунды 3), потом как только появилась обложка, вновь встало все на свои места (кнопки + и - не работали корректно).
Пробовал удалять интеграцию, ставить заново, ставить более раннюю версию, ставить последнюю версию, сбрасывать колонки и заново добавлять в яндекс. Даже с чистого листа на Виртуальную машину устанавливал HA и интеграцию яндекс станции.. ничего не помогло.

**Comment by Opontios** [comment]
@KAPACb1988
```service: media_player.volume_set``` срабатывает, но потом в какой-то момент снова сбрасывается на 50. А вот если использовать 
`extra:
volume_level:`
 то после каждой фразы сразу сбрасывается на 50. 
Это давно уже началось, как видите по дате создания ветки, и для себя после всех этих танцев с бубном я тоже не смог определить с чем это связано. Думал что обновление прошивок колонок, но так как не следил за тем, какие версии были до этого, с уверенностью не мог сделать вывод. Но потом эта версия потерпела крах, потому что из 4-х колонок (2 с часами, 1 лайт и одна старая мини) симптомы сначала проявились на 2-х с часами и на лайт, потом подключилась к ним и старая мини, а потом лайт вернулась в нормальное состояние и так и до сих пор остается. Так что только автор интеграции может нас просветить.

**Comment by Opontios** [comment]
С сегодняшнего дня состояние изменилось с точностью до наоборот. "Станция мини 2 с часами" заработала нормально и перестала работать "станция лайт".

**Comment by KAPACb1988** [comment]
У меня сейчас две станции мини 2 без часов нормально стали работать, еще и мультируме стало хорошо синхронизироваться, раньше не помню, но что то не так было, в lovelace один плеер не показывал то что надо, и кнопки плохо работали. А сейчас даже в мультируме, инфа о песне и дорожка длительности и громкости хорошо синхронизируется на 2ух плеерах сразу, на любом нажимаешь кнопку следующий трек, и опять на 2ух плеерах инфа о песне меняется, все правильно синхронизирует и кнопки все работают (даже в мультируме).

**Comment by Opontios** [comment]
Да, уже дня три как все заработало как и должно на всех колонках. Остается непонятным с чем это было связано.

**Comment by Sergmasharov** [comment]
У меня вообще перестал работать tts на макс с часами, как ни бейся не говорит. Изредка помогают перезагрузки, плюс она не меняет громкость уведомлений. На каком уровне была, так и говорит.
Пользуюсь вот этим способом:
`  alice_custom_volume2:
    sequence:
    - service: media_player.play_media
      entity_id: media_player.yandex_station  # замените на вашу колонку
      data:
        media_content_id: Внимание! Важное сообщение...
        media_content_type: text  # поддерживается text и dialog
        extra:
          volume_level: 0.8`
